github/provider.go

```diff
@@ -56,6 +57,12 @@ func Provider() terraform.ResourceProvider {
 				Default:     0,
 				Description: descriptions["read_delay_ms"],
 			},
+			"parallel_requests": {
```

Comment:
By default, does this make all requests parallel? Is there a limit to the number of requests that can be made in parallel?

Reply:
The default for the flag is to keep request serial for backward consistency and to be safe when calling github.com without changing it.
In regards of the upper limit, I am allowing terraform to work "normally" and use the go routines chosen by the user through the `--parallelism` flag that is documented [here](https://developer.hashicorp.com/terraform/cli/commands/apply#parallelism-n); default is 10. The locking system was preventing that to comply with github.com rate limits.

Reply:
Understood, thank you!